translator/tocwconfig/sampleConfig/container_insights_jmx.json

```diff
@@ -0,0 +1,41 @@
+{
+  "agent": {
+    "debug": true
+  },
+  "logs": {
+    "metrics_collected": {
+      "emf": { },
+      "kubernetes": {
+        "cluster_name": "TestCluster",
+        "jmx_container_insights": true
+      }
+    }
+  },
+  "metrics": {
```

Comment:
Remove this metrics section. There's no reason to have it for this test.

Reply:
We want the test to be invalid, should we do something else?

Reply:
I'm not sure I follow. How does this make this test invalid?